config/config.go

```diff
@@ -107,6 +107,42 @@ func NewStoreConfig() (*Store, error) {
 	return storeConfig, nil
 }
 
+func (oc *OpenRegistryConfig) Validate() error {
+	if oc == nil {
+		return fmt.Errorf("invalid config, cannot be nil")
+	}
+	v := validator.New()
+
+	english := en.New()
+	uni := ut.New(english, english)
+	trans, _ := uni.GetTranslator("en")
```

Comment:
@guacamole shouldn't we be checking these errors?

Reply:
you're right